metrics/cli/test/services/common/auth_cli_test.dart

```diff
@@ -0,0 +1,38 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:cli/services/common/cli/auth_cli.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("AuthCli", () {
+    test(
```

Comment:
✅ 
Should we also test that the auth cli adds the specified auth argument to the beginning or end of the arguments list? Also, that it composes argument and value correctly?